organization-workflow.md

```diff
@@ -0,0 +1,65 @@
+# Organization workflow
+
+The general workflow to organize a session is as follows:
+
+## 2 weeks before
+
+- Create the next paper candidates issue.
+
+    The corresponding issue template is almost good to go: it just
+    requires setting the date of the next session in the title.
+
+- Archive last session and create the new one.
+
+    PR a change to
+    [src-d/reading](https://github.com/src-d/reading-club) club to put
+    the previous session in the [Past papers](README.md#past-papers)
+    section with the template
+
+        - __YYYY.MM.DD__ [Title of the paper](url of the paper on GDrive). ([notes](Url of the notes on GDrive))
+    
+    And create the new session with the template
+
+        - __YYYY.MM.DD__ at __4PM CET__: [TBD](url of the candidates issues)
+
+## 1 week before
+
+Create a poll to decide on the next paper.
+
+The poll should be created on the #reading-club community channel. It
+uses the poll widget:
+
+    /poll "What papers would you like to study during next session, on the xxth of xxxxx?" "Title1: url1" "Title2: url2"
+
+## 5 days before
+
+- Pick the winner.
+
+- Add the PDF to the [GDrive
+  directory](https://drive.google.com/drive/u/1/folders/1cAzkDW_sXb49gRZvvhVvXu3Wi_84lQdn).
+
+- Create the notes document.
+
+    When we have time we link the authors to their respective
+    homepages and include relevant links (code, data, etc).
+
+- PR a change to the next session in the
+  [readme](README.md#next-sessions).
+
+    We use the following template:
+
+        - __YYYY.MM.DD__ at __4PM CET__: [Title of the paper](url of the pdf on GDrive). We take notes in a [public GDoc](url of the notes on GDrive), you are very welcome to ask questions or give thoughts in it!
```

Comment:
Usually, as soon as winner is picked (or PR is merged) - we also announce that on Slack #reading-club